File: functionTests/repmatTests/test3.R
.T_repmat <- function(v,n,p){ #A@5 WHAT IS THIS???????????????????????
  if (p==1){M = cbind(rep(1,n)) %*% v} #A@5 a matrix of column of v
  else { M = matrix(rep(v,n),n,(length(v)*p),byrow=T)} # removed cat("!");
  M
}

v = c(1,2,3,4)

res = .T_repmat(v, 2, 3)
print(res)

File: functionTests/repmatTests/test3.py
import numpy as np

def _T_repmat(v, n, p):
    M = np.c_[np.repeat(1, n)]@np.atleast_2d(v)
    M = np.tile(M, p)

    return M

v = np.array([1,2,3,4])

res = _T_repmat(v, 2, 3)
print(res)